core/src/main/java/org/sonarsource/sonarlint/core/client/api/connected/ConnectedAnalysisConfiguration.java

```diff
@@ -53,9 +51,7 @@ public String toString() {
   private String generateString() {
     var sb = new StringBuilder();
     sb.append("[\n");
-    if (projectKey != null) {
-      sb.append("  projectKey: ").append(projectKey).append("\n");
-    }
+    sb.append("  projectKey: ").append(projectKey).append("\n");
```

Comment:
Nitpick: the last line could also be chained

Reply:
:+1: